From c46cb01c60d16c3187e62a04aa76874bd3e2fe44 Mon Sep 17 00:00:00 2001
From: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Date: Fri, 11 Oct 2024 10:00:06 +0200
Subject: [PATCH 1325/1695] dt-bindings: add HSI/HSE/MSI Kernel clock bindings
 for STM32MP21

HSI/HSE/MSI Kernel clocks are used in low power mode for some peripherals.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 164083a995ec33cc6e98a43973663a54ec97248a dt-bindings: add HSI/HSE/MSI Kernel clock bindings for STM32MP21]
Signed-off-by: Gabriel Fernandez <gabriel.fernandez@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/406045
Change-Id: I92369b392b184ab036138d590edd8a465e1e4602
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/410784
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Reviewed-by: Etienne CARRIERE <etienne.carriere@foss.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 core/include/dt-bindings/clock/st,stm32mp21-rcc.h | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/core/include/dt-bindings/clock/st,stm32mp21-rcc.h b/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
index 1b6d533e1..04d7d5f35 100644
--- a/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
+++ b/core/include/dt-bindings/clock/st,stm32mp21-rcc.h
@@ -310,8 +310,11 @@
 #define CK_DAP			287
 #define CK_KER_ETR		288
 #define CK_KER_STM		289
+#define HSI_KER_CK		290
+#define HSE_KER_CK		291
+#define MSI_KER_CK		292
 
-#define STM32MP21_LAST_CLK	290
+#define STM32MP21_LAST_CLK	293
 
 #define CK_SCMI_ICN_HS_MCU	0
 #define CK_SCMI_ICN_SDMMC	1
@@ -426,5 +429,8 @@
 #define CK_SCMI_BUS_STM		110
 #define CK_SCMI_KER_STM		111
 #define CK_SCMI_KER_ETR		112
+#define CK_SCMI_HSI_KER_CK	113
+#define CK_SCMI_HSE_KER_CK	114
+#define CK_SCMI_MSI_KER_CK	115
 
 #endif /* _DT_BINDINGS_STM32MP21_CLKS_H_ */
-- 
2.47.2

